charts/stable/common/templates/lib/controller/_pod.tpl

```diff
@@ -23,9 +23,15 @@ hostNetwork: {{ . }}
 {{- with .Values.hostname }}
 hostname: {{ . }}
 {{- end }}
+{{- if .Values.dnsPolicy }}
 {{- with .Values.dnsPolicy }}
 dnsPolicy: {{ . }}
 {{- end }}
```

Comment:
```suggestion
{{- if .Values.dnsPolicy }}
 dnsPolicy: {{ .Values.dnsPolicy }}
```
Because of the added `if`, the `with` became redundant.

Reply:
yeah that would be fine too :)